.babelrc

```diff
@@ -27,6 +27,17 @@
           }
         ]
       ]
+    },
+    "DYNAMIC_CONFIG_TEST": {
+      "plugins": [
+        [
+          "./lib/plugin.js",
+          {
+            "config": "<%= CONFIG %>",
```

Comment:
Also I think here and in documentation is better to use `${CONFIG}` syntax as it like as javascript template literals. (This `<%= CONFIG%>` syntax is something from grunt ;-) path templates)